llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -36,6 +37,11 @@
 #include <memory>
 
 using namespace llvm;
+using llvm::util::sycl::cl_intel_devicelib_assert;
+using llvm::util::sycl::cl_intel_devicelib_complex;
+using llvm::util::sycl::cl_intel_devicelib_complex_fp64;
+using llvm::util::sycl::cl_intel_devicelib_math;
+using llvm::util::sycl::cl_intel_devicelib_math_fp64;
```

Comment:
somehow my previous comment on this disappeared. Please don't `using` all the individual enum elements.

Reply:
BTW, I see that you add merge commits when you implement review comments:
> Merge remote-tracking branch 'upstream/sycl' into link_fallback_devic… …

This does not seem right to me. Maybe @bader can comment more. I think this can lead to missing review comments. You should only use git push/git pull when working with the review, so that `jinge90:link_fallback_devicelib_on_demand` does not have merge commits. Only if you have conflicts, you should rebase your feature branch. I do it like this:

`git rebase <hash> --onto intel_llvm/sycl`

where <hash> is the commit right below the first feature branch commit in `git log` output.

Reply:
Hi, @kbobrovs 
Thank you for the remind.  I updated patch to use c++11 scoped enum for DeviceLibExt, the definition looks like:
enum class DeviceLibExt : std::uint32_t {cl_intel_devicelib_assert,.....}
By doing this, we don't need to "using" all individual enum elements, we only need to "using llvm::util::sycl::DeviceLibExt;" in source code and use "DeviceLibExt::cl_intel_devicelib_*" to refer to all DeviceLibExt enum elements. Is this OK to you?

Reply:
Yes, thank you.